backend/uclapi/dashboard/views.py

```diff
@@ -156,12 +156,21 @@ def get_started(request):
         request.session["user_id"]
     except KeyError:
         logged_in = False
+    articles = get_articles()
+
+
 
     token = get_temp_token()
     return render(request, 'getStarted.html', {
         'initial_data': {
             'temp_token': token,
-            'logged_in': str(logged_in)
+            'logged_in': str(logged_in),
+            'url1': articles[0][0],
```

Comment:
yes i can, let me stick that in now!

Reply:
<3
We could also perhaps have a constant somewhere where we specify exactly how many articles we want to download? It might be that in the future we have a different frontend layout where having links to more than three articles would work well so it would be good to centralise this parameter.

Reply:
yeh that would probably be good!

Reply:
@ChristopherHammond13 Stuck the constant in for you!